private_gpt/settings/settings_loader.py

```diff
@@ -16,7 +16,7 @@
 _settings_folder = os.environ.get("PGPT_SETTINGS_FOLDER", PROJECT_ROOT_PATH)
 
 # if running in unittest, use the test profile
-_test_profile = ["test"] if "tests.fixtures" in sys.modules else []
+_test_profile = ["test"] if "unittest" in sys.modules else []
```

Comment:
Don't revert this change. It was done on purpose to fix an issue with latests versions of llamaindex. Check d3acd85fe34030f8cfd7daf50b30c534087bdf2b

Reply:
Hm. That's odd. I don't remember making that change at all, there's no reason I would have needed to.  I wonder if I got something wrong when I committed the patch.  Easy enough to revert...